### ISSUE TITLE
Update the documentation of this project

### ISSUE THREAD

**Issue by Vedant-Jayesh-Oza** [Gdsc-Lbce/GDSC_LBCE_Website#52]
Hey @ashish-ad, since the project is in react, there are no guidelines in the documentation regarding how to set up the react project into the local system. So if you feel this is a issue needs to be fixed then assign the issue to me I will do some necessary changes in the readme file.

**Comment by ashish-ad** [comment]
yeah sure @Vedant-Jayesh-Oza, u can do it make sure to include the .md file to react file.
